README.md

```diff
@@ -1 +1,10 @@
-# angular-filter-track-by-lab
\ No newline at end of file
+# Using track by inside ng-repeat and $filter in Controller
+
+## Objectives
+
+- Refactor our ng-repeat to use a Controller filter
+- Integrate "track by" clause to enhance DOM render change performance
+
+## Instructions
+
+Refactor the `ng-repeat` in this repo to use track by. Also, apply a filter on this list in the controller, filtering on the search property.
```

Comment:
Convert these instructions into a list - this will be easier for the students to read.